CHANGELOG.md

```diff
@@ -1,9 +1,39 @@
 # Unreleased
 
+# 9.13.0 (Feature release)
+
+## New features
+
+### Make Sass errors clearer to users
+
+Previously, the Sass app continued to run even after Sass stopped compiling because of an error.
+
+Now, when there's a Sass error, it creates a blank `application.css` file to make the site look broken to users.
+
+If you fix the Sass error, the site will automatically reload.
+
+We have also changed the Gulp log level to be quieter, so that errors stand out.
+
+This was added in [Pull request #990: Make Sass errors clearer to the user] (https://github.com/alphagov/govuk-prototype-kit/pull/990).
+
+### Opt in to the new GOV.UK Frontend link styles
+
+Links now have underlines that are consistently thinner and a bit further away from the link text.
+
+Links also have a clearer hover state, where the underline gets thicker to make the link stand out to users.
+
+The new link styles are opt-out by default because [Chromium browsers currently cannot display them](https://github.com/alphagov/govuk-frontend/issues/2204).
```

Comment:
```suggestion
The new link styles are opt-in by default because [Chromium browsers currently cannot display them](https://github.com/alphagov/govuk-frontend/issues/2204).
```